jerry-core/parser/js/js-parser-util.c

```diff
@@ -167,18 +167,11 @@ parser_flush_cbc (parser_context_t *context_p) /**< context */
 #ifdef PARSER_DUMP_BYTE_CODE
   if (context_p->is_show_opcodes)
   {
-    const char *name_p;
-
-    if (PARSER_IS_BASIC_OPCODE (context_p->last_cbc_opcode))
-    {
-      name_p = cbc_names[context_p->last_cbc_opcode];
-    }
-    else
-    {
-      name_p = cbc_ext_names[PARSER_GET_EXT_OPCODE (context_p->last_cbc_opcode)];
-    }
-
-    JERRY_DEBUG_MSG ("  [%3d] %s", (int) context_p->stack_depth, name_p);
+    JERRY_DEBUG_MSG ("  [%3d] %s",
+                     (int) context_p->stack_depth,
+                     PARSER_IS_BASIC_OPCODE (context_p->last_cbc_opcode)
+                     ? cbc_names[context_p->last_cbc_opcode]
+                     : cbc_ext_names[PARSER_GET_EXT_OPCODE (context_p->last_cbc_opcode)]);
```

Comment:
I realized this coding style is new for me. It is usually:
```
(a ? b
   : c)
```
Is it frequently used? In only found something like this in `re-compiler.c` but that looks even more strange.